.drone.yaml

```diff
@@ -0,0 +1,80 @@
+---
+kind: pipeline
+name: linux-amd64
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: ci
+  image: briandowns/rancher-build-base:v0.1.1
+  commands:
+  - ./scripts/ci
+  when:
+    event:
+    - push
+    - pull_request
+    - tag
+
+- name: build
+  image: briandowns/rancher-build-base:v0.1.1
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - sleep 20
```

Comment:
What does the sleep do?
CC: @rmweir

Reply:
yea, confused why this would be needed

Reply:
It looked on race condition, but I removed it